queue/queue.go

```diff
@@ -22,6 +25,18 @@ const (
 
 type Store struct {
 	*bolt.DB
+	pubsub.Publisher
+}
+
+func (db *Store) publishResponseJSON(resp mdm.Response) error {
+	buf := new(bytes.Buffer)
+	enc := json.NewEncoder(buf)
```

Comment:
hmm, imo with the `/connect` responses the value is that you get the raw feed directly from the device. 

Right now we don't actually have all the possible responses from mdm, so if you exposed the raw plist coming directly to `/connect` the user would probably be able to take better advantage of it.